### ISSUE TITLE
Three extra options

### ISSUE THREAD

**Issue by GSoftwareDevelopment** [Honatas/bootstrap-4-autocomplete#14]
Hello.

This component is really easy to use and expand, well... In my repo i have expanded version of this component.
I add three configurable options. Maybe you like it :+1:

The first is the ability to add a class to the _dropdown-menu_ item created by the component. This enables the creation of e.g. a scrolling drop-down list.
Option: **class**
Default: (empty string)

The second option is to allow a string comparison from the beginning. Uses the _startsWith_ method to compare, not _indexOf_ :) Simple, but very useful.
Option: **startWith**
Default: false

The third option is a **fetchSource** callback function to get autocomplete data.
In my fork, I have included a broader description and an example of use (is a bit stupid: P but shows, how to use it)
Option: **fetchSource**

I also added the ability to display all possible matching answers by setting the **maximumItems** value to 0 (zero). Useful when we use the class setting for dropdown-menu, e.g .:
CSS
```CSS
.autocomplete {
  height:200px;
  overflow-y:auto;
}
```
JS
```JS
    $('#input').autocomplete({
        source: src,
        highlightClass: 'text-danger',
        treshold: 1,
        maximumItems: 0,
        class: 'autocomplete'
    });
```

**Comment by Honatas** [comment]
Hi @GSoftwareDevelopment.

Thanks for the interest and time spent creating this PR.

I really enjoyed these additions. Apart from the fetchSource option which I have decided not to implement (as explained in #5), I'll definitely take a time to review and merge all the other features as soon as possible.

**Comment by Honatas** [comment]
Just released v1.3.0 which implements some of these suggestions.
